internal/commands/groups.go

```diff
@@ -2,110 +2,25 @@ package commands
 
 import (
 	"encoding/json"
-	"strings"
 
+	featureFlagsConstants "github.com/checkmarx/ast-cli/internal/constants/feature-flags"
 	commonParams "github.com/checkmarx/ast-cli/internal/params"
+	"github.com/checkmarx/ast-cli/internal/services"
 	"github.com/checkmarx/ast-cli/internal/wrappers"
-	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
-const accessManagementEnabled = "ACCESS_MANAGEMENT_ENABLED" // feature flag
-
-func createGroupsMap(groupsStr string, groupsWrapper wrappers.GroupsWrapper) ([]*wrappers.Group, error) {
```

Comment:
all groups related logic suppose to be in a group service and not under project service, we need to keep the same separation

Reply:
maybe all this group file suppose to be a "service" or a domain inside project

Reply:
done